unique_toolkit/unique_toolkit/app/verification.py

```diff
@@ -11,12 +12,16 @@ class WebhookVerificationError(Exception):
     pass
 
 
+T = TypeVar("T")
+
+
 def verify_signature_and_construct_event(
     headers: dict[str, str],
     payload: bytes,
     endpoint_secret: str,
     logger: logging.Logger = logging.getLogger(__name__),
-):
+    event_constructor: Callable[..., T] = Event,
```

Comment:
I think instead of `Callable[,..., T] = Event` this is better as a type of event like so

`event_constructor: Event  = Event` since all other event type are a subclass of the event, no?

Reply:
Not necessarily the case. The events are starting to get very diverse. And as far as the code is concerned, there is no reason to limit it to Event type.